### ISSUE TITLE
Pop!_Shop Can't install Dbeaver Community id's are not valid

### ISSUE THREAD

**Issue by deyvisonrocha** [pop-os/beta#109]
**How did you upgrade to 20.04? (Fresh install / Upgrade)**

 
Fresh install
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

Pop!_Shop

**Issue/Bug Description:**

I can't install the DBeaver Community from Pop!_Shop

**Steps to reproduce (if you know):**

1. Open **Pop!_Shop**
1. Search DBeaver
1. Click on the install button
1. Error message:
![image](https://user-images.githubusercontent.com/686067/80487924-f54ca600-8933-11ea-83d6-c801e883f9c9.png)

**Expected behavior:**

Install it!

**Other Notes:**

**Comment by brs17** [comment]
This has been resolved.

**Comment by tcoker** [comment]
I encountered the same issue today. What is the resolution?

**Comment by leviport** [comment]
It was solved with an update, so you shouldn't have run into it. Does running `sudo apt update && sudo apt upgrade` fix it for you?

**Comment by tcoker** [comment]
No, I continue to get the same result. System is pop 20.04 upgraded from 19.10, all up to date.

![image](https://user-images.githubusercontent.com/4139561/97049147-d0329c00-1540-11eb-9b9a-232360f6479f.png)

**Comment by ifaniqbal** [comment]
I got same issue on pop os 21.04, Failed to install “DBeaver Community”, The package id's '' are not valid

**Comment by leviport** [comment]
I can confirm that this is happening again. 

As a workaround, the flatpak version works fine.
